cloudformation/stack.yaml

```diff
@@ -1409,8 +1409,9 @@ Resources:
         ZipFile: |
           import json
           import boto3
-          from botocore.vendored import requests
+          import urllib3
```

Comment:
Where do these dependencies come from? Does something else run `pip install urllib3`?

Reply:
Also good question. Lambda functions come with this installed. So we're good here too. :)